src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  currentKey: string,
```

Comment:
```suggestion
  currentPressedKey: string,
```
Readabily is everything :)